uploadNeuroDB/NeuroDB/MRIProcessingUtility.pm

```diff
@@ -1387,7 +1387,7 @@ sub dicom_to_minc {
         # We need to use quotemeta here to make sure each character of a given variable is interpreted
         # as is and not as a regex special character
         # Note: '$\'\t\'' is a TAB for Unix grep
-        my $pattern = join('$\'\t\'', quotemeta($studyUID), quotemeta($seriesUID), quotemeta($echo), quotemeta($seriesDesc));
+        my $pattern = join('\t', quotemeta($studyUID), quotemeta($seriesUID), quotemeta($echo), quotemeta($seriesDesc));
```

Comment:
@nicolasbrossard do you agree with those changes? I remember you changed that section due to a bug in https://github.com/aces/Loris-MRI/commit/a38df855043c3833d7a94ea00df1f0d858f0e5e5